mlir/lib/Conversion/MIGraphXToTosa/MIGraphXToTosa.cpp

```diff
@@ -235,9 +235,38 @@ LogicalResult ConvConverter<ConvType>::matchAndRewrite(
   newShape.push_back(outShape[1]);
   Type newOutTy = RankedTensorType::get(newShape, outputTy.getElementType());
 
+  auto expandTo2D = [&rewriter, loc](mlir::Value value) {
```

Comment:
[stylistic] nit : I generally would use a lambda if it interacts heavily with variables in the scope where as in this one only uses rewriter and loc. I d consider defining a function with proper return types and some docs.